diff --git a/debian/jellyfin-ffmpeg8.install b/debian/jellyfin-ffmpeg8.install
index 00f41ac..a82e5b1 100644
--- a/debian/jellyfin-ffmpeg8.install
+++ b/debian/jellyfin-ffmpeg8.install
@@ -7,3 +7,4 @@ libavformat/libavformat.so* usr/lib/jellyfin-ffmpeg/lib
 libavutil/libavutil.so* usr/lib/jellyfin-ffmpeg/lib
 libswresample/libswresample.so* usr/lib/jellyfin-ffmpeg/lib
 libswscale/libswscale.so* usr/lib/jellyfin-ffmpeg/lib
+debian/tmp/usr/lib/jellyfin-ffmpeg/include
diff --git a/debian/rules b/debian/rules
index 6a4ffca..3e2d0ec 100755
--- a/debian/rules
+++ b/debian/rules
@@ -30,6 +30,7 @@ CONFIG := --prefix=${TARGET_DIR} \
 	--enable-libdrm \
 	--enable-libxml2 \
 	--enable-libass \
+	--enable-libaribcaption \
 	--enable-libfreetype \
 	--enable-libfribidi \
 	--enable-libfontconfig \
@@ -98,6 +99,7 @@ override_dh_auto_clean:
 	dh_auto_clean || true
 
 override_dh_auto_install:
+	$(MAKE) install-headers DESTDIR=$(CURDIR)/debian/tmp PREFIX=${TARGET_DIR}
 
 override_dh_usrlocal:
 
diff --git a/docker-build-win64.sh b/docker-build-win64.sh
index 6525679..1c9e41e 100755
--- a/docker-build-win64.sh
+++ b/docker-build-win64.sh
@@ -172,6 +172,23 @@ meson setup fontconfig fontconfig_build \
 meson configure fontconfig_build
 ninja -j$(nproc) -C fontconfig_build install
 
+# LIBARIBCAPTION
+git clone -b v1.1.2 --depth=1 https://github.com/xqq/libaribcaption.git
+pushd libaribcaption
+mkdir build
+pushd build
+cmake \
+    -DCMAKE_TOOLCHAIN_FILE=${FF_CMAKE_TOOLCHAIN} \
+    -DCMAKE_INSTALL_PREFIX=${FF_DEPS_PREFIX} \
+    -DCMAKE_BUILD_TYPE=Release \
+    -DBUILD_SHARED_LIBS=OFF \
+    -DARIBCC_SHARED_LIBRARY=OFF \
+    ..
+make -j$(nproc)
+make install
+popd
+popd
+
 # HARFBUZZ
 git clone -b 14.2.1 --depth=1 https://github.com/harfbuzz/harfbuzz.git
 meson setup harfbuzz harfbuzz_build \
@@ -606,6 +623,7 @@ fi
     --enable-libfontconfig \
     --enable-libharfbuzz \
     --enable-libass \
+    --enable-libaribcaption \
     --enable-libbluray \
     --enable-libmp3lame \
     --enable-libopus \
diff --git a/docker-build.sh b/docker-build.sh
index 3e1682f..6f90068 100755
--- a/docker-build.sh
+++ b/docker-build.sh
@@ -127,6 +127,28 @@ prepare_extra_common() {
     echo "fontconfig/libfontconfig.so* usr/lib/jellyfin-ffmpeg/lib" >> ${DPKG_INSTALL_LIST}
     popd
 
+    # LIBARIBCAPTION
+    pushd ${SOURCE_DIR}
+    git clone -b v1.1.2 --depth=1 https://github.com/xqq/libaribcaption.git
+    pushd libaribcaption
+    mkdir build
+    pushd build
+    cmake \
+        ${CMAKE_TOOLCHAIN_OPT} \
+        -DCMAKE_INSTALL_PREFIX=${TARGET_DIR} \
+        -DCMAKE_BUILD_TYPE=Release \
+        -DBUILD_SHARED_LIBS=ON \
+        -DARIBCC_SHARED_LIBRARY=ON \
+        -DARIBCC_USE_FREETYPE=ON \
+        -DARIBCC_USE_FONTCONFIG=ON \
+        ..
+    make -j$(nproc) && make install && make install DESTDIR=${SOURCE_DIR}/libaribcaption
+    echo "libaribcaption${TARGET_DIR}/lib/libaribcaption.so* usr/lib/jellyfin-ffmpeg/lib" >> ${DPKG_INSTALL_LIST}
+    echo "libaribcaption${TARGET_DIR}/include/aribcaption usr/lib/jellyfin-ffmpeg/include" >> ${DPKG_INSTALL_LIST}
+    popd
+    popd
+    popd
+
     # HARFBUZZ
     pushd ${SOURCE_DIR}
     git clone -b 14.2.1 --depth=1 https://github.com/harfbuzz/harfbuzz.git
